File: cpp/ABC166/166_B.cpp
#include <iostream>
#include <map>
using namespace std;

int main() {
    int N, K;
    std::map<int,int> snuke;

    cin >> N >> K;

    for (int i=0; i < K; i++) {
        int d;
        cin >> d;

        for (int j=0; j < d; j++) {
            int tmp;
            cin >> tmp;
            snuke[tmp-1]++;
        }
    }

    int ans = 0;
    for (int i=0; i < N; i++) {
        if (snuke[i] == 0) {
            ans++;
        }
    }
    cout << ans << endl;
}

File: python/ABC/ABC166/166_B.py
N, K = map(int, input().split())

snuke = {}
for i in range(N):
    snuke[i] = 0

for i in range(K):
    d = int(input())
    A = list(map(int, input().split()))

    for j in range(d):
        snuke[A[j]-1] += 1

ans = 0
for i in range(N):
    if snuke[i] == 0:
        ans += 1

print(ans)